libs/ui/src/pure/InlineBanner/index.tsx

```diff
@@ -144,17 +184,42 @@ export function InlineBanner({
   children,
   className,
   hideIcon,
-  bannerType,
+  bannerType = 'information',
   borderRadius,
   orientation,
   iconSize,
   iconPadding = '0',
+  customIcon,
   padding,
   margin,
   width,
+  dismissable,
+  bannerID,
 }: InlineBannerProps) {
-  const effectiveBannerType = bannerType || 'alert'
-  const colorEnums = getColorEnums(effectiveBannerType)
+  const colorEnums = getColorEnums(bannerType)
+  const LOCAL_STORAGE_KEY = dismissable ? `${bannerID}` : undefined
```

Comment:
We usually reserve all caps for constants. Not the case here as this is a local variable.

Reply:
Will fix

Reply:
fixed